doc/azure/azure-osx-template.yml

```diff
@@ -0,0 +1,27 @@
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    matrix:
```

Comment:
I think it's worth extending these matrices a bit further

Reply:
I have expanded the matrixes a bit further. Hoping that this is sufficient to demonstrate.